Cargo.toml

```diff
@@ -1,25 +1,25 @@
 [package]
 name = "schnorr"
-version = "0.6.0"
+version = "0.7.0-rc.0"
```

Comment:
Don't update the version now please. Do it in a separated commit in the `release-0.7` branch once this gets merged.